CHANGELOG.md

```diff
@@ -17,6 +18,9 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 - Add jumping to a reply inside a thread when opening a thread from a thread reply in the channel view [#2647](https://github.com/GetStream/stream-chat-swift/pull/2647)
 - Add jumping to a reply inside a thread when opening a channel from a reply as the given message around id [#2647](https://github.com/GetStream/stream-chat-swift/pull/2647)
 
+### 🐞 Fixed
+- Allow editing messages with attachments [#2659](https://github.com/GetStream/stream-chat-swift/pull/2659)
```

Comment:
This is not a fix, but an addition, can we move to the ✅  Added section?

Reply:
I guess it could be subjective here, but we shipped the SDK without this capability on purpose (It was not an accident), so that is why I consider this a "Missing Feature" and not a "Bug". Hence the idea of adding it to the "Added" section. If we add it to the "Fixed" section, it could give a bad impression that we "forgot" about this or that we were not aware of this. 

Either way, if we keep it in the "Fixed" section, at least it sounds reasonable to re-word it to "Fix not being able to edit message attachments", since the "Allow" prefix looks like an addition, and not a fix 🤔